osu.Game/Screens/Edit/Setup/LabelledFileChooser.cs

```diff
@@ -35,7 +38,27 @@ internal class FileChooserLabelledTextBox : LabelledTextBoxWithPopover, ICanAcce
         [Resolved]
         private OsuGameBase game { get; set; } = null!;
 
-        public FileChooserLabelledTextBox(params string[] handledExtensions)
+        private readonly BindableWithCurrent<string> current = new BindableWithCurrent<string>();
+
+        public Bindable<string> Current
+        {
+            get => current.Current;
+            set => current.Current = value;
+        }
```

Comment:
This is still weird. It's a string, it's "current", but it's pretty useless. It's steered entirely by `currentFile` and is serving as a glorified event.

Either make the `currentFile` bindable public, or just make this an event or something, I'd say.

Reply:
Didn't notice the `currentFile` bindable, can use it as the type of the current and rely on it instead of string.